runtime/parser2/expression.go

```diff
@@ -401,13 +402,15 @@ func defineInvocationExpression() {
 					}
 					p.next()
 					expectArgument = true
+
 				case lexer.TokenParenClose:
 					endPos = p.current.EndPos
 					p.next()
 					atEnd = true
-					break
```

Comment:
Just curious why this was removed?

Reply:
It's not needed, cases don't fall through in Go